coq/coq.el

```diff
@@ -160,6 +160,16 @@ See also option `coq-hide-additional-subgoals'."
   :type '(choice regexp (const nil))
   :group 'coq)
 
+(defcustom coq-show-proof-stepwise nil
+  "Display the proof terms stepwise in the *response* buffer.
+  This option can be combined with option `coq-diffs'.
+  It is mostly useful in three window mode, see also
+  `proof-three-window-mode-policy' for details."
```

Comment:
You should probably remove the leading spaces here:

```suggestion
This option can be combined with option `coq-diffs'.
It is mostly useful in three window mode, see also
`proof-three-window-mode-policy' for details."
```

(otherwise the output of <kbd>C-h v coq-show-proof-stepwise RET</kbd> is weird)